utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pln3.cpp

```diff
@@ -835,6 +838,56 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 37:
+        {
+            test_case_name = "crop";
+
+            // Uncomment to run test case with an xywhROI override
+            for (i = 0; i < images; i++)
+            {
+                roiTensorPtrSrc[i].xywhROI.xy.x = 0;
+                roiTensorPtrSrc[i].xywhROI.xy.y = 0;
+                roiTensorPtrSrc[i].xywhROI.roiWidth = 100;
+                roiTensorPtrSrc[i].xywhROI.roiHeight = 180;
+            }
+
+            // Uncomment to run test case with an ltrbROI override
+            /*for (i = 0; i < images; i++)
```

Comment:
It is better to pass parameters through arguments instead of commenting and uncommenting code.